### ISSUE TITLE
[Bug] (or might be a feature request) How to format text inside a graphic component?

### ISSUE THREAD

**Issue by ankitjangir01** [apache/echarts#20075]
### Version

5.5.0

### Link to Minimal Reproduction

https://echarts.apache.org/examples/en/editor.html?code=PYBwLglsB2AEC8sDeAoWszGAG0iAXMmuiaWbAMYwBmE0ApoWAE4Cu9x5pAvgDSdd0AEwCGYEQGd6YQqkGCJwVswqNYAbQHyy6gEQBhMfQDmwZgE9dvWLoCSEiewn6l0MFZsBlegDd6zCDBzTxBsQN0AXX5tQT0AQQkhakTqIQ8AFgBmdOskXRFdQmzuKK0Y9D1DMHoAJgyAVgA2AEZc_MLYbOtdACMOrpsKDvT0kujynQMjTIaWtoKinJs-xe6hwhGxsvLKo3TZ1uR21eX-pd112E3Sicmq-nqD-bPulc7zy-vx24qp6vTGk8jgt3q8XoNhqMbj9fvcAOxAvIgga9cEXSFbGGwowATkRx1BpxO6I2UO-P121RxQjS1nSTUOSLRbxRnzJ2xilPo1PxyPOLI-GOhtwiHJ45IU_gg9AkhE0MLkWPQQRAagA5CBpWqJT8IEJCBqtTrbswREIIKxZRo1fV6gBSbWwNWNe1q4VY1Rufxym0ABgd1j9DvdMPo0CoQjUiqVpEC9AAtgA5ETx9X3UwWbVin4-ETYdgG-yOGUuVhuLMx2B8bPaBMgAAWkggVujldg1GAFEtBqk2Goapr8mrlewIh69GwBoHMOHMRDXGMpobEAoskHpBV6uqAA8wBWfth6NQZE7PdVmPvbpgCKew-fLxMJEFD2vKzuT-vOZ_bmqkCIAD7ZJktrcA-bYkGqzbFhI07gbAEQAHQAFbAHQAAUaoADrQGqACU35kMaEzvnEYTGNABpnv4YE_B2bieBAABeajNDkBGkGEDAABL0BAxj1ieNQ1PwACQYlidodFgAA6rx_EyPUvq-qJ4kiZJMBgAAYimEDYOYBoAAr5hIsAAFIiAA1iIzDiLAngiNAMFEeUADuepgPWBrNEpIDbrBtzcIOgXcAA3EAA

### Steps to Reproduce

Just visit the link 

### Current Behavior

The rich text is being rendered as plain text inside a graphic>text.

### Expected Behavior

The text should be formatted as rich text properties are given.

### Environment

```markdown
- OS: Windows 10
- Browser: Chrome
- Framework: React, TS
```

### Any additional comments?

Basically I want to render the text as in this screenshot-
![image](https://github.com/apache/echarts/assets/93923854/9ad2f541-8947-4d23-9391-e735c6ba619e)

But all I am getting is unformatted text inside chart (refer reproduction link).

**Comment by helgasoft** [comment]
see [official example](https://echarts.apache.org/examples/en/editor.html?c=line-graphic)
